src/shadowbox/server/api.yml

```diff
@@ -27,7 +27,34 @@ paths:
               examples:
                 '0':
                   value: >-
-                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234}
+                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234, "version": "1.0.0"}
+  /server/hostname:
```

Comment:
I'm confused about changing the server IP - what are the semantics of the operation? I would have expected that we check that the hostname resolves to the management IP - what happens to the API endpoint after chaning the server IP?

Relatedly, if we want to support IP addresses, I think "host" would be a more appropriate name, as "hostname" implies a domain name.

Reply:
**Semantic**
This is changing the host for access keys. This is not actually changing the server IP or domain. It's just making it possible to change what the manager sends to users. Currently they manually edit the string, which is an unnecessary pain.

If this is not clear, we can clarify the API. Maybe `host-for-access-keys`? OR maybe all we need is to clarify the API documentation.

**Naming**
I agree that "host" would be clearer. However, we already use `hostname` throughout the code, so we may want to use that for consistency. We can also change everything to `host`, provided we ensure backward-compatibility.

There are many places that use both terms interchangeably (and the [location browser api](https://developer.mozilla.org/en-US/docs/Web/API/Location) actually uses host for hostname+port!), so it may not be a huge deal to keep hostname.